chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/CertificationDateReviewer.java

```diff
@@ -18,7 +17,14 @@ public CertificationDateReviewer(ErrorMessageUtil msgUtil) {
     }
     @Override
     public void review(CertifiedProductSearchDetails listing) {
-        if (listing.getCertificationDate() > new Date().getTime()) {
+        if (listing.getCertificationDate() == null && StringUtils.isEmpty(listing.getCertificationDateStr())) {
+            listing.getErrorMessages().add(msgUtil.getMessage("listing.certificationDateMissing"));
+        } else if (listing.getCertificationDate() == null && !StringUtils.isEmpty(listing.getCertificationDateStr())) {
+            listing.getErrorMessages().add(msgUtil.getMessage("listing.badCertificationDate", listing.getCertificationDateStr()));
+        }
+
+        if (listing.getCertificationDate() != null && listing.getCertificationDate() > System.currentTimeMillis()) {
+            //TODO: print out the certification date
```

Comment:
This is a newly added TODO, right? Should we do something with it?

Reply:
I don't remember. I think I wanted to add the parsed certification date to the error message so I added the TODO but later thought this is an existing error message that users have been used to seeing (assuming anyone has ever gotten this error) so I decided not to change it. I'm going to remove the TODO.